scripts/nix-profile.fish.in

```diff
@@ -0,0 +1,36 @@
+if test -n "$HOME" && test -n "$USER"
+
+    # Set up the per-user profile.
+    # This part should be kept in sync with nixpkgs:nixos/modules/programs/shell.nix
```

Comment:
This file does not exist anymore.

Reply:
I'll remove it from the other files too!